samples/msal-node-silent-flow/index.js

```diff
@@ -0,0 +1,71 @@
+/*
```

Comment:
If we are not doing auth code as part of the sample, there is not need to have an express app. You can just show silent flow in a command line app (similar to the device code/refresh flow sample), without all of the extra overhead.

Reply:
I will be adding auth-code soon. Checked in just the happy case.